### ISSUE TITLE
[Accessibility]A11y_VSCode_ViewExtensions_ScreenReader: VoiceOver does not read the necessary headings on the screen.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52398]
**Environment Details:**
VSCode Version : 1.24.0
OS Version :MACOS High Sierra 10.13.5 Beta (17F70a)
Additional Details:
ScreenReader,Voiceover,MAS1.3.1
**Repro Steps:**
1. Launch VS Code.
2. Navigate to Activity Bar and select "View Extensions "(Cntrl+Shift+X) button.
3. Navigate to complementary landmark -"View Extensions " section and read the content "View Extensions" using basic navigation.

**Actual:**
Screenreader does not read the content "View Extensions" as appopriate heading which is so.

**Expected:**
Screenreader should read- "View Extensions" as heading with appropriate heading level.

**Recommendations:**
Add appropriate native heading level tag to the content "View Extensions".
or,
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
The screenreader users will not be able to know the appropriate headings present on the screen.

**MAS Reference:**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}
Does this issue occur when all extensions are disabled?: Yes
**Attachment For Reference:**
[[Accessibility]A11y_VSCode_ViewExtensions_ScreenReader_Voiceover2.pptx](https://github.com/Microsoft/vscode/files/2118129/Accessibility.A11y_VSCode_ViewExtensions_ScreenReader_Voiceover2.pptx)

**Comment by AccessibilityTestingTeam-TCS** [comment]
Please check the screenshot,Voiceover does not read the "Extension" as heading level.
![accessibility a11y_vscode_viewextensions_screenreader](https://user-images.githubusercontent.com/39204909/41893743-6a743454-793a-11e8-88a2-a4017f821002.png)

**Comment by AccessibilityTestingTeam-TCS** [comment]
Basically the point here is that the headings like "Search", "Explorer" etc. are not identified as headings. They should have proper heading levels. The side bar content - "Search", "Explorer" etc. can be accessed using landmarks (D command in NVDA for navigating to landmarks) and then try to read headings inside that using (H command in NVDA for headings). There are no heading levels found for above mentioned headings.

![headings](https://user-images.githubusercontent.com/39204909/42144796-b28bce10-7dda-11e8-8113-0a4ac822cb16.png)


Hope this explains the needed.

**Comment by isidorn** [comment]
Dup of https://github.com/Microsoft/vscode/issues/51760
